src/main/kotlin/com/vito/sanel/Runner.kt

```diff
@@ -1,11 +1,70 @@
 package com.vito.sanel
 
+import com.vito.sanel.models.MemberType
+import java.time.Duration
+import java.time.LocalDateTime
+import kotlin.math.exp
+import kotlin.random.Random
+
 class Runner {
 
     companion object {
-        @JvmStatic fun main(args : Array<String>) {
-            println("Ok")
+        @JvmStatic
+        fun main(args: Array<String>) {
+            val start = LocalDateTime.now()
+            println("Starting at: $start")
+            generateBoard()
+            println("Finished in: " + Duration.between(start, LocalDateTime.now()).seconds + " sec")
         }
-    }
 
+        private fun generateBoard() {
+            var temperature = INITIAL_TEMPERATURE
+            var solution = false
```

Comment:
nit: solution is a noun, so it is better to use something else. Maybe `solved` or `isSolved`, `solutionFound`?